plugins/NBBC/class.nbbc.plugin.php

```diff
@@ -219,7 +219,8 @@ function DoURL($bbcode, $action, $name, $default, $params, $content) {
    public function Format($Result) {
       $Result = str_replace(array('[CODE]', '[/CODE]'), array('[code]', '[/code]'), $Result);
       $Result = $this->NBBC()->Parse($Result);
-      return $Result;
+      $Mixed = html_entity_decode($Result);
+      return Gdn_Format::html($Mixed);
```

Comment:
I apologies for the extra changes. I did not realize that my psr-2 fixer plugin had kicked in on this file. It makes it a little difficult to zero in on the changed code. I added this comment to highlight the change for you guys.